### PR TITLE
Nerfed NG Payout significantly to ensure balance 

### PR DESCRIPTION
**Pull request by rhydiaan**
I was getting lucky and was making more money than anticipated so I've nerfed payouts to reduce ROI.

### CODE REVIEW

**Review by SamTipper — APPROVED** [review]
Looks good b0ss

File: judge/bot.py [hunk 1]
```diff
@@ -42,7 +42,6 @@ async def help_embed(self, ctx, type):
         )
 
         if type == "rps":
-            embed.add_field(name='$rps start [Gameid]', value='Starts a game of rock paper scissors with the specified ID.', inline=False)
```
Comment (Coombszy) [inline judge/bot.py:45]: @rhydiaan Why was this removed?

**Review by Coombszy — APPROVED** [review]
